src/app/d2-loadout-builder/routes.ts

```diff
@@ -1,10 +1,12 @@
 import { ReactStateDeclaration } from '@uirouter/react';
 import LoadoutBuilder from './LoadoutBuilder';
 
-export const states: ReactStateDeclaration[] = [
-  {
-    name: 'destiny2.loadoutbuilder',
-    component: LoadoutBuilder,
-    url: '/loadoutbuilder'
-  }
-];
+export const states: ReactStateDeclaration[] = $featureFlags.d2LoadoutBuilder
```

Comment:
Was also thinking we could keep the loadout builder accessible so that people who are eager to try it can point their url to it (and don't need to spin up a dev server.)

Reply:
That’s fair. Perhaps we could lazy load it though so it doesn’t add code weight to the initial page load.